test/examples/verbose.ml

```diff
@@ -25,6 +25,18 @@ let f () = if x then
   else y
 
 
+(* No Flagging here - only 2 match levels deep *)
+let f () =
+  let l = [] in
+  begin match l with
+    | [] ->
+      begin match l with
+        | [] -> true
+        | _ -> false
+      end
+    | _ -> true
+  end
+
```

Comment:
Added this test case here, which ended up adjusting the line numbers of the violations in `test/test.ml`.